### DISCUSSION TITLE
CUSTOM THEMES

### DISCUSSION THREAD

**Discussion by TT-ReBORN** [TT-ReBORN/Georgia-ReBORN#99] [Show and tell]
## In this thread you can share your custom themes so everybody can enjoy them!

Create a new post here and rename the `georgia-reborn-custom.jsonc` you want to share to `georgia-reborn-custom.jsonc.zip`,
drag and drop it into your post. This step is only required for Github so you can directly upload it here, because otherwise 
these file types won't be accepted when uploading. You can of course upload it somewhere else and share the link in your post.
Once you have downloaded posted custom configs, you just need to rename and remove the `.zip` from the file name and 
copy it to your config directory: `foobar2000\profile\georgia-reborn\configs`. 
Start or reload foobar and select the new custom themes via top menu Options > Theme > Custom.

## Since the release of Beta 11, you can create your own custom themes.

**Options > Theme > Custom > Edit custom theme**
First select a custom theme slot you want to edit, all 5 custom themes have a predefined custom theme.
When a custom theme slot is selected, you can use the context menu > right click > Edit custom theme for quick navigation.
You can either select the color via the color picker or paste a HEX value in the input field.
It will apply all changes in real time and saves it automatically in the georgia-reborn-custom.jsonc config file.
Each color has a name that you can also find in the georgia-reborn-custom.jsonc config file and modify it there.
To reset the colors to the default ones, select the "Reset" option from the drop down menu.

[Tip: Download the resource pack from the Github page to open the custom theme template and modify colors in 
  Photoshop or Gimp.](https://github.com/TT-ReBORN/Georgia-ReBORN-resources/raw/master/Resources/Georgia-ReBORN_Resources.7z)

**Comment by TT-ReBORN** [comment]
## Violet play ( Custom Theme 01 ):
![Georgia_ReBORN_Custom_Theme_01](https://raw.githubusercontent.com/TT-ReBORN/Georgia-ReBORN-resources/master/Custom-Themes/Georgia_ReBORN_Custom_Theme_01.jpg)

## Gray plasma ( Custom Theme 02 ):
![Georgia_ReBORN_Custom_Theme_02](https://raw.githubusercontent.com/TT-ReBORN/Georgia-ReBORN-resources/master/Custom-Themes/Georgia_ReBORN_Custom_Theme_02.jpg)

## Electric teal ( Custom Theme 03 ):
![Georgia_ReBORN_Custom_Theme_03](https://raw.githubusercontent.com/TT-ReBORN/Georgia-ReBORN-resources/master/Custom-Themes/Georgia_ReBORN_Custom_Theme_03.jpg)

## Arctic pearl ( Custom Theme 04 ):
![Georgia_ReBORN_Custom_Theme_04](https://raw.githubusercontent.com/TT-ReBORN/Georgia-ReBORN-resources/master/Custom-Themes/Georgia_ReBORN_Custom_Theme_04.jpg)

## Pink noise ( Custom Theme 05 ):
![Georgia_ReBORN_Custom_Theme_05](https://raw.githubusercontent.com/TT-ReBORN/Georgia-ReBORN-resources/master/Custom-Themes/Georgia_ReBORN_Custom_Theme_05.jpg)

## Golden snow ( Custom Theme 06 ):
![Georgia_ReBORN_Custom_Theme_06](https://raw.githubusercontent.com/TT-ReBORN/Georgia-ReBORN-resources/master/Custom-Themes/Georgia_ReBORN_Custom_Theme_06.jpg)

## Brown amber ( Custom Theme 07 ):
![Georgia_ReBORN_Custom_Theme_07](https://raw.githubusercontent.com/TT-ReBORN/Georgia-ReBORN-resources/master/Custom-Themes/Georgia_ReBORN_Custom_Theme_07.jpg)

## Walnut butter ( Custom Theme 08 ):
![Georgia_ReBORN_Custom_Theme_08](https://raw.githubusercontent.com/TT-ReBORN/Georgia-ReBORN-resources/master/Custom-Themes/Georgia_ReBORN_Custom_Theme_08.jpg)

## Dark desert ( Custom Theme 09 ):
![Georgia_ReBORN_Custom_Theme_09](https://raw.githubusercontent.com/TT-ReBORN/Georgia-ReBORN-resources/master/Custom-Themes/Georgia_ReBORN_Custom_Theme_09.jpg)

## Black dusk ( Custom Theme 10 ):
![Georgia_ReBORN_Custom_Theme_10](https://raw.githubusercontent.com/TT-ReBORN/Georgia-ReBORN-resources/master/Custom-Themes/Georgia_ReBORN_Custom_Theme_10.jpg)

## Download and remove the .zip from the filename, copy it to your `foobar2000\profile\georgia-reborn\configs` directory.
[georgia-reborn-custom.zip](https://github.com/user-attachments/files/16629178/georgia-reborn-custom.zip)



Start or reload foobar and select the new custom themes via top menu Options > Theme > Custom.
Enjoy 10 new custom themes and also use the custom presets if you like via top menu Options > Style > Presets > Custom.

-TT

**Comment by d3rb** [comment]
hi tom i've been very busy lately and just now i saw that i can create my own theme. Thanks for this really really cool feature. I edited theme01 and now the question, how can I edit the boot logo / reload screen / backround and loading bar

edit:

Hello Tom under Main/Bar is .VolumbarFrame and .ProgressBarFrame editable, for Timeline not is it possible to add this option?

greetings
Enrico

**Reply by TT-ReBORN** [reply]
Hi Enrico,

glad you like the new feature. The reason why the startup screen colors can not be set via config, is because of technical limitation, 
it needs to load first all scripts and colors to be able to use them. Also if you change the background color, the logo would not look good on other background colors. The user would also need to create a new logo or modify the colors of the existing logo. Or replace the original logo with his own logo. Well you could change the colors for the startup screen, but only in the javascript startup screen code as it is of now.

Timeline frame can be added.

-Tom

**Comment by legacy9899** [comment]
Hello, so my georgia-reborn folder doesn't have configs folder, so I create it and put the jsonc file inside the folder but the custom theme not showed in Option > Theme, what should I do? Also thx for your work, it's really great

**Reply by TT-ReBORN** [reply]
Hi, 
you're using a very old version from the Github page, that is why it's not working. 
Make a new portable installation and [download the latest version from the Beta thread here](https://github.com/TT-ReBORN/Georgia-ReBORN/discussions/74#discussion-4190799). 
Everything will return to normal once the Beta is finished, which will be soon!

-TT
